security/tailscale/src/opnsense/service/templates/OPNsense/Tailscale/rc.conf.d

```diff
@@ -0,0 +1,47 @@
+# DO NOT EDIT
+# THIS FILE IS AUTOMATICALLY GENERATED - ANY CHANGES WILL BE OVERWRITTEN
+#
+{% if helpers.exists('OPNsense.tailscale.settings.enabled') and OPNsense.tailscale.settings.enabled|default("0") == "1" %}
```

Comment:
```suggestion
{% if not helpers.empty('OPNsense.tailscale.settings.enabled')  %}
```

a little bit less code, same applies to some of the others below

Reply:
Awesome, thank you for that. Sorry, not familiar with jinja